Source/WebCore/PAL/pal/spi/cocoa/CoreMaterialSPI.h

```diff
@@ -36,14 +36,31 @@
 
 #else
 
+typedef NSString * MTCoreMaterialVisualStyleCategory NS_STRING_ENUM;
+
+extern MTCoreMaterialVisualStyleCategory const MTCoreMaterialVisualStyleCategoryStroke;
+extern MTCoreMaterialVisualStyleCategory const MTCoreMaterialVisualStyleCategoryFill;
+
+typedef NSString * MTCoreMaterialVisualStyle NS_STRING_ENUM;
+
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStyleNone;
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStylePrimary;
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStyleSecondary;
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStyleTertiary;
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStyleQuaternary;
+extern MTCoreMaterialVisualStyle const MTCoreMaterialVisualStyleSeparator;
+
 typedef NSString * MTCoreMaterialRecipe NS_STRING_ENUM;
 
+extern MTCoreMaterialRecipe const MTCoreMaterialRecipeNone;
 extern MTCoreMaterialRecipe const MTCoreMaterialRecipePlatformChromeLight;
 extern MTCoreMaterialRecipe const MTCoreMaterialRecipePlatformContentUltraThinLight;
 extern MTCoreMaterialRecipe const MTCoreMaterialRecipePlatformContentThinLight;
 extern MTCoreMaterialRecipe const MTCoreMaterialRecipePlatformContentLight;
 extern MTCoreMaterialRecipe const MTCoreMaterialRecipePlatformContentThickLight;
 
+extern NSDictionary <NSString *, id> *MTVisualStylingCreateDictionaryRepresentation(MTCoreMaterialRecipe, MTCoreMaterialVisualStyleCategory, MTCoreMaterialVisualStyle, NSDictionary <NSString *, id> *);
```

Comment:
there shouldn't be a space before the <NSString, right?

Reply:
This is how it is in the actual private header, which is probably the only acceptable reason to have it this way.